src/LobbyForm.tsx

```diff
@@ -12,96 +13,16 @@ type Props = Readonly<{
 function LobbyForm({ socket }: Props): React.ReactElement {
   return (
     <div className={css(hostStyles.form)}>
-      <Host socket={socket} />
-      <div className={css(hostStyles.or)}>- or -</div>
-      <Join socket={socket} />
-    </div>
-  );
-}
-
-type HostProps = Readonly<{
-  socket: Socket;
-}>;
-
-function Host({ socket }: HostProps): ReactElement {
-  function onHost(e: FormEvent<HTMLButtonElement>): void {
-    e.preventDefault();
-    socket.emit("join_lobby", { lobbyID: null });
-  }
-
-  return (
-    <div className={css(hostStyles.row)}>
-      <div className={css(hostStyles.text)}>Create a new game</div>
-      <div>
-        <button onClick={onHost} className={css(hostStyles.button)}>
-          &gt;
-        </button>
-      </div>
-    </div>
-  );
-}
-
-function Join({ socket }: HostProps): ReactElement {
-  const [lobbyID, onChangeLobbyID] = useFormActions();
-
-  function onJoinLobby(e: FormEvent<HTMLButtonElement>): void {
-    e.preventDefault();
-    if (lobbyID.length < 3) {
-      console.warn("lobby id is probably more than 3 characters");
-      return;
-    }
-    socket.emit("join_lobby", { lobbyID });
-  }
-
-  return (
-    <div className={css(hostStyles.row)}>
-      <div>
-        <input
-          className={css(hostStyles.input)}
-          placeholder="Join a lobby using a name"
-          onChange={onChangeLobbyID}
-          value={lobbyID}
-        />
-      </div>
-      <div>
-        <button onClick={onJoinLobby} className={css(hostStyles.button)}>
-          &gt;
-        </button>
-      </div>
+      <Host socket={socket} onClick={() => {}} />

```

Comment:
This onclick is just an empty function, seems kinda pointless, let's remove this unless there's a specific reason you need to pass an empty onClick to Host